support/ovscon2025/index.html

```diff
@@ -130,8 +33,749 @@ <h2>How to attend</h2>
+<pre>
+OVS checksum offloads API and TSO support for the userspace datapath
+was introduced in multiple steps over several years and has grown more
+complex with the addition of tunnel support recently.
+The close ties with the DPDK checksum offloads API resulted in complex
+operations in OVS packet processing. And here complex means bugs were
+present.
+
+This talk will present the recent revamp of this API: from the initial
+fixes and the unit tests coverage additions, to the new API, its
+enhanced readbility and the (hopefully) easier maintenance such as
+better isolation of the API from the DPDK and Linux kernel offloads
+API.
+
+As part of this talk, performance numbers with some popular nics will
+be discussed.
+</pre>
+
+<h3 id="t21">You Want to Upgrade WHAT? A Field Guide to Risky Network Changes <a href="#day2">|TOP|</a></h3>
+<h3>Speaker(s): Nadia Pinaeva, NVIDIA; Dumitru Ceara, Red Hat; Patryk Diak, Red Hat</h3>
+<pre>
+Every active project accumulates technical debt. For OVN-Kubernetes, our Layer2
+network topology, a result of incremental additions and unfortunate design
+choices, had hit its limit. Not only did it make different topologies support
+more complicated than it should be, it also imposed feature limitations for
+EgressIP, Virtual Machines live migration and Network Interconnect. A
+re-architecture was unavoidable.
+
+This session details how we leveraged the OVN Transit Router to redesign our L2
+network from the ground up for stability and scale. But the real story isn’t
+just the new design: having customers already using this feature in production
+introduced an interesting set of network engineering challenges to solve. If
+you have never:
+
+  * Connected the same switch and router with 2 links
+  * Copied the same MAC on multiple ports
+  * Generated dummy IPs just to fix routing
+  * Hand-crafted Route Advertisements to remove stale routes
+
+Join our session and follow along on our topology upgrade adventure!
+</pre>
+
+<h3 id="t22">Space - building a CMS with OVN Concepts <a href="#day2">|TOP|</a></h3>
+<h3>Speaker(s): Felix Huettner, StackIt</h3>
+<pre>
+When our existing OpenStack Neutron deployment could no longer meet
+new requirements, we decided to build an additional CMS based on OVN
+design concepts.
+
+In this talk we will share how we adapted these concepts to our
+implementation of "spaced".
+
+We will walk through the core components of our system: a new
+"spacebound" OVSDB for defining resources and "spaced" that acts as a
+custom northd. Spaced translates our high-level requirements into the
+standard OVN northbound database.  It thereby integrates with the Resources
+that OpenStack Neutron is still managing in northbound.
+
+If you were hoping that this talk is actually about running OVN in
+space, then sorry for making you excited.
+</pre>
+
+<h3 id="t23">Programming OVS bridges using OVN Bridge Controller <a href="#day2">|TOP|</a></h3>
+<h3>Speaker(s): Numan Siddique, Crusoe-AI</h3>
```

Comment:
Not sure about the dash.  In the original email from Numan there was just a space.
The rest looks good.